src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1639,8 +1666,8 @@ void TorrentHandle::handleSaveResumeDataAlert(const libtorrent::save_resume_data
     libtorrent::entry &resumeData = useDummyResumeData ? dummyEntry : *(p->resume_data);
     if (useDummyResumeData) {
         resumeData["qBt-magnetUri"] = toMagnetUri().toStdString();
-        resumeData["qBt-paused"] = isPaused();
-        resumeData["qBt-forced"] = isForced();
+        resumeData["paused"] = isPaused();
+        resumeData["auto_managed"] = isAutoManaged();
```

Comment:
`isAutoManaged()` doesn't exist in 4.1.x. The direct call is `m_nativeStatus.auto_managed`